lib/slack-ruby-bot/hooks/hello.rb

```diff
@@ -1,15 +1,20 @@
 module SlackRubyBot
   module Hooks
     class Hello
-      attr_accessor :logger
+      attr_accessor :logger, :connected_at
 
       def initialize(logger)
         self.logger = logger
       end
 
       def call(client, _data)
         return unless client && client.team
-        logger.info "Successfully connected team #{client.team.name} (#{client.team.id}) to https://#{client.team.domain}.slack.com."
+        logger.info "Successfully #{@connected_at ? 'reconnected' : 'connected'} team #{client.team.name} (#{client.team.id}) to https://#{client.team.domain}.slack.com."
+
+        connected_at = Process.clock_gettime(Process::CLOCK_MONOTONIC)
+        logger.info "Time elapsed since last connection: #{connected_at - self.connected_at} seconds" if self.connected_at
```

Comment:
I run a lot of bots where this would double output. Let's change the message above to include this? How about "Successfully ... after #{conencted_at - self.connected_at}s"?

Reply:
Whoop, didn't read this properly, will edit the message